File: _REPO/APRESS/distributed-.net-programming-in-vb-.net/Chapter08/ObjectPooling/SimpleCarClient/CarClient.vb
Imports SimpleCarLibrary

Module CarClient

   Sub Main()

      SomeClientMethod()
      Console.ReadLine()
      
   End Sub
   
   Sub SomeClientMethod
      Dim carSvc As New CarService()
      carSvc.GetInfo()
      carSvc.AnotherMethod()   
   End Sub   
End Module

File: _REPO/APRESS/distributed-.net-programming-in-csharp/Chapter08/ObjectPooling/SimpleCarClient/CarClient.cs
using System;
using SimpleCarLibrary;
using System.Runtime.Remoting;

namespace SimpleCarClient
{
   class CarMain
   {
      static void Main(string[] args)
      {
         SomeClientFunction();
                  
         Console.ReadLine();
      }

      static void SomeClientFunction()
      {     
         CarService carSvc = new CarService();
         Console.WriteLine(carSvc.GetInfo());
         carSvc.AnotherMethod();
      }

   }
}
